ansible/roles/stack-sunbird/defaults/main.yml

```diff
@@ -300,6 +302,27 @@ kong_limit_memory: 256M
 kibana_oauth_authenticated_email_domains: []
 
 ##### livenessprobe and readinessprobe #######
+
+dhiti_liveness_readiness:
+  healthcheck: false
```

Comment:
Why are we making the health check false?

Reply:
Because on that Service, Currently Health Check is not enabled over /ping path

Reply:
Is this planned to be implemented? If yes, this is fine. If not, you can change to `tcpSocket`

Reply:
Yes, Developer will enable the health check on /path

Reply:
Alright then!